interface/therapy_groups/therapy_groups_views/groupDetailsParticipants.php

```diff
@@ -192,7 +192,7 @@
                 { "width": "35%", "targets": 5 }
             ],
             "pageLength":6,
-            "order": [[ 0, "asc" ],[ 2, "desc" ]],
+            "order": false,
```

Comment:
lets put a comment here that this wasn't working, so did the sorting in the php code.